tests/test_fft.py

```diff
@@ -2,51 +2,75 @@
 import numpy as np
 
 from aspire import config
-from aspire.utils.numeric import fft_class
+from aspire.utils.numeric import fft_object
 
 fft_backends = ['scipy', 'pyfftw']
-numeric_classes = [np, np]
+numeric_modules = [np, np]
 
 # Create Cupy fft backend if Cupy is enabled and lib exits.
 if config.common.cupy:
     try:
         import cupy as cp
         fft_backends.append('cupy')
-        numeric_classes.append(cp)
+        numeric_modules.append(cp)
     except ImportError:
         pass
 
-fft_classes = [fft_class(backend) for backend in fft_backends]
+fft_objects = [fft_object(backend) for backend in fft_backends]
 
 
 class ConfigTest(TestCase):
     def testFft(self):
-        for fft, xp in zip(fft_classes, numeric_classes):
+        for fft, xp in zip(fft_objects, numeric_modules):
             for nworkers in (-1, 1, 2):
                 a = xp.random.random(100)
                 b = fft.fft(a, workers=nworkers)
                 c = fft.ifft(b, workers=nworkers)
                 self.assertTrue(xp.allclose(a, c))
 
     def testFft2(self):
-        for fft, xp in zip(fft_classes, numeric_classes):
+        for fft, xp in zip(fft_objects, numeric_modules):
             for nworkers in (-1, 1, 2):
                 a = xp.random.random((100, 100))
                 b = fft.fft2(a, workers=nworkers)
                 c = fft.ifft2(b, workers=nworkers)
                 self.assertTrue(xp.allclose(a, c))
 
     def testFftn(self):
-        for fft, xp in zip(fft_classes, numeric_classes):
+        for fft, xp in zip(fft_objects, numeric_modules):
             for nworkers in (-1, 1, 2):
                 a = xp.random.random((50, 50, 50))
                 b = fft.fftn(a, axes=(0, 1, 2), workers=nworkers)
                 c = fft.ifftn(b, axes=(0, 1, 2), workers=nworkers)
                 self.assertTrue(xp.allclose(a, c))
 
     def testShift(self):
-        for fft, xp in zip(fft_classes, numeric_classes):
+        for fft, xp in zip(fft_objects, numeric_modules):
             a = xp.random.random(100)
             b = fft.ifftshift(a)
             c = fft.fftshift(b)
             self.assertTrue(xp.allclose(a, c))
+
+    def testCenteredFft(self):
+        for fft, xp in zip(fft_objects, numeric_modules):
+            for nworkers in (-1, 1, 2):
+                a = xp.random.random(100)
+                b = fft.centered_fft(a, workers=nworkers)
```

Comment:
Verify that the center sample of `b` value is equal to the sum of `a`. Same for 2D and 3D.

Reply:
Done in commit 7763477.